test.js

```diff
@@ -23,3 +23,26 @@ test('main', t => {
 
 	t.pass();
 });
+
+test('preventRepeat', t => {
+	const random = uniqueRandom(1, 10, true);
+	let count = 1000;
+	let currentValue;
+	let previousValue;
+
+	while (--count > 0) {
+		currentValue = random();
+
+		if (
+			currentValue === previousValue ||
+			currentValue > 10 ||
+			currentValue < 1
+		) {
+			t.fail();
+		}
+
+		previousValue = currentValue;
+	}
+
+	t.pass();
+});
```

Comment:
You need a test that actually ensures it does what it does. From what I can tell, this is just a copy of the previous test.